sources/web/datalab/polymer/test/settings-manager-test.html

```diff
@@ -20,9 +20,9 @@
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <script src="../../web-component-tester/browser.js"></script>
 
-    <script src="../modules/SettingsManager.js"></script>
+    <script src="../modules/settings-manager/settings-manager.js"></script>
```

Comment:
Shouldn't this now be a link statement pointing to setting-manager.html? And then not have the script statement on line 26?
Likewise in timeout-manager-test.